app/helpers/application_helper.rb

```diff
@@ -76,4 +85,11 @@ def confirmable_action(label, object, options = {}, &blk)
 
     render partial: "/confirmable_action", locals: options.merge(label: label, object: object)
   end
+
+  def read_file(path, limit:)
```

Comment:
I wasn't able to test these output file previews. Any hint on how to get smaller outputs in GWAS?